[2025-05-01T21:16:31.267+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-01T21:16:31.327+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline_demo.transform_data_1 manual__2025-05-01T21:16:23.234657+00:00 [queued]>
[2025-05-01T21:16:31.341+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline_demo.transform_data_1 manual__2025-05-01T21:16:23.234657+00:00 [queued]>
[2025-05-01T21:16:31.343+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-01T21:16:31.375+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data_1> on 2025-05-01 21:16:23.234657+00:00
[2025-05-01T21:16:31.382+0000] {standard_task_runner.py:63} INFO - Started process 405 to run task
[2025-05-01T21:16:31.390+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline_demo', 'transform_data_1', 'manual__2025-05-01T21:16:23.234657+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpmt3c2q7w']
[2025-05-01T21:16:31.398+0000] {standard_task_runner.py:91} INFO - Job 18: Subtask transform_data_1
[2025-05-01T21:16:31.444+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-01T21:16:31.505+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline_demo.transform_data_1 manual__2025-05-01T21:16:23.234657+00:00 [running]> on host b3f0d5cbc6df
[2025-05-01T21:16:31.696+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline_demo' AIRFLOW_CTX_TASK_ID='transform_data_1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T21:16:23.234657+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-01T21:16:23.234657+00:00'
[2025-05-01T21:16:31.698+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-01T21:16:31.757+0000] {logging_mixin.py:188} INFO - Tache transform_data bien executee
[2025-05-01T21:16:31.758+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-01T21:16:31.759+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-01T21:16:31.773+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline_demo, task_id=transform_data_1, run_id=manual__2025-05-01T21:16:23.234657+00:00, execution_date=20250501T211623, start_date=20250501T211631, end_date=20250501T211631
[2025-05-01T21:16:31.807+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-01T21:16:31.823+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
